api/scpca_portal/test/management/commands/test_sync_batch_jobs.py

```diff
@@ -66,7 +66,7 @@ def test_sync_batch_jobs(self, mock_batch_get_jobs):
         # Set up mock for get_jobs
         mock_response = [{"jobId": job.batch_job_id} for job in self.jobs]
         # All AWS Batch job statuses (7) + FAILED & terminated job (1)
-        mock_response[0]["status"] = "SUBMITTED"
+        mock_response[0]["status"] = "PROCESSING"
```

Comment:
Shouldn't this have parity with https://docs.aws.amazon.com/batch/latest/userguide/job_states.html

Reply:
Oh, sorry, this was an accidental change - fixed. Thank you!